Sources/Runner/Commands/Runner.swift

```diff
@@ -176,3 +189,47 @@ func runDanger(logger: Logger) throws {
     // Return the same error code as the compilation
     exit(proc.terminationStatus)
 }
+
+private func bootstrapArgs() -> [String] {
+    #if !os(Linux)
+    return []
+    #else
+    // The Linux bots require extra arguments in order to locate the corelibs.
+    // We can potentially drop this by installing some stable linux toolchain
+    // after Swift gets ABI and module stability.
+    //
+    // Compute if SwiftPM is bootstrapping.
+    let env = ProcessInfo.processInfo.environment
+    guard env.keys.contains("SWIFTPM_BOOTSTRAP") else { return [] }
+    guard let buildPath = env["SWIFTPM_BUILD_DIR"],
+        !buildPath.isEmpty else {
+        return []
+    }
+    
+    // Construct the required search paths relative to the build directory.
+    let libdir = buildPath.appending(".bootstrap/lib/swift/linux")
+    let incdir = libdir.appending("/x86_64")
+    let dispatchIncdir = incdir.appending("/dispatch")
+    
+    return [
+        "-I\(incdir)",
+        "-I\(dispatchIncdir)",
+        "-L\(libdir)",
+        "-Xcc", "-F\(incdir)",
+    ]
+    #endif
+}
+
+private func getSwiftPMManifestArgs(swiftPath: String) -> [String] {
+    // using "xcrun --find swift" we get
+    // /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc
+    // we need to transform it to something like:
+    // /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/4_2
+    let fileManager = FileManager.default
+    
+    Logger().logInfo(swiftPath)
+    
+    let swiftPMDir = swiftPath.replacingOccurrences(of: "bin/swiftc", with: "lib/swift/pm/4_2")
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 233;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)